paddle/framework/backward.h

```diff
@@ -27,6 +27,8 @@ extern std::unique_ptr<OperatorBase> Backward(
     const OperatorBase& forwardOp,
     const std::unordered_set<std::string>& no_grad_vars);
 
+// TODO(someone): Add target as parameter and generate backward op
```

Comment:
Let's assign this TODO to some specific person when we create it.

Reply:
done